packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterCookieManager.java

```diff
@@ -15,17 +15,15 @@
 import io.flutter.plugin.common.MethodChannel.Result;
 
 class FlutterCookieManager implements MethodCallHandler {
+  private final MethodChannel methodChannel;
 
-  private FlutterCookieManager() {
-    // Do not instantiate.
-    // This class should only be used in context of a BinaryMessenger.
-    // Use FlutterCookieManager#registerWith instead.
+  FlutterCookieManager(BinaryMessenger messenger) {
+    methodChannel = new MethodChannel(messenger, "plugins.flutter.io/cookie_manager");
+    methodChannel.setMethodCallHandler(this);
   }
 
   static void registerWith(BinaryMessenger messenger) {
```

Comment:
I guess we could completely remove this method...

Reply:
I guess so, done.